modules/permission-management/src/Volo.Abp.PermissionManagement.Application.Contracts/Volo/Abp/PermissionManagement/Integration/IPermissionIntegrationService.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Volo.Abp.Application.Services;
+
+namespace Volo.Abp.PermissionManagement.Integration;
+
+[IntegrationService]
+public interface IPermissionIntegrationService : IApplicationService
+{
+    Task<List<PermissionGrantOutput> > IsGrantedAsync(List<PermissionGrantInput> input);
```

Comment:
```suggestion
    Task<List<PermissionGrantOutput>> IsGrantedAsync(List<PermissionGrantInput> input);
```

Reply:
Instead of directly returning a `List`, return a `ListResultDto`. This will be easier for us to add more properties to the output object without breaking existing clients.